microsoft-teams/installation.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Microsoft Teams'
+---
+
+**Never miss another message from a customer in Microsoft Teams**
+
+<Warning>
+    Microsoft Teams integration is currently in beta. The integration won't currently be available for your workspace until we've enabled the feature flag.
```

Comment:
I think `we've enabled the feature flag` is a bit too developer-focused. Perhaps we can do `until we've enabled it for your workspace`